Description: Allows for the enumeration of many additional ports, including ttyACM for Arduino Unos
Author: Scott Howard <showard314@gmail.com>

Index: rxtx/src/gnu/io/RXTXCommDriver.java
===================================================================
--- rxtx.orig/src/gnu/io/RXTXCommDriver.java	2010-10-04 23:14:47.236148507 -0400
+++ rxtx/src/gnu/io/RXTXCommDriver.java	2010-10-04 23:18:56.976108303 -0400
@@ -574,20 +574,7 @@
 					if(osName.equals("Linux"))
 					{
 						String[] Temp = {
-						"ttyS", // linux Serial Ports
 						"ttySA", // for the IPAQs
-						"ttyUSB", // for USB frobs
-						"rfcomm",       // bluetooth serial device
-						"ttyircomm", // linux IrCommdevices (IrDA serial emu)
-						};
-						CandidatePortPrefixes=Temp;
-					}
-					else if(osName.equals("Linux-all-ports"))
-					{
-					/* if you want to enumerate all ports ~5000
-					   possible, then replace the above with this
-					*/
-						String[] Temp = {
 						"comx",      // linux COMMX synchronous serial card
 						"holter",    // custom card for heart monitoring
 						"modem",     // linux symbolic link to modem.
